include/ua_plugin_securitypolicy.h

```diff
@@ -178,32 +178,34 @@ typedef struct
     const size_t encryptingBlockSize;
 } UA_SecurityPolicySymmetricModule;
 
+/**
+ * This struct defines initialization Data that is potentially needed by every security policy.
+ * Some of these, like the revocation list for example, may be omitted (empty ByteString).
+ */
+typedef struct
+{
+    UA_ByteString localPrivateKey;
+    UA_ByteString localCertificate;
+    UA_ByteString certificateTrustList;
+    UA_ByteString certificateRevocationList;
+} UA_Endpoint_SecurityContext_RequiredInitData;
+
 struct UA_Endpoint_SecurityContext {
+    /**
+     * \brief Creates a new endpoint context with the supplied initialization data.
+     *
+     * \param securityPolicy the securityPolicy this function is invoked on.
+     * \param initData the required initialization data for the endpoint context.
+     */
     UA_StatusCode (*const newContext)(const UA_SecurityPolicy *securityPolicy,

```

Comment:
In order to not go full-circle with the refactorings...
How do you feel about renaming the `endpointContext` to `policyContext`?
That should not require any changes to the current context...
But make it an abstraction that is applicable to many endpoints by the client.